third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -295,6 +295,8 @@ resource "google_composer_environment" "test" {
 		node_count = 4
 
 		software_config {
+			image_version = "composer-1.7.0-airflow-1.10.2"
```

Comment:
Ah sorry I didn't catch this earlier- this also needs beta-only guards, since without it the test in the GA provider will destroy/create instead of doing an update.

Reply:
Thanks, updated.